scss/misc/_delta-buttons.scss

```
@@ -17,7 +18,7 @@ button.delta-button-round {
     border-color: rgba(206, 217, 224, 0.5);
     background-image: none;
     cursor: not-allowed;
-    color: rgba(92, 112, 128, 0.6);
+    color:rgb(51, 49, 49);
```

```suggestion
    color: var(--globalText);  
```

I think using this variable is a better fit, as it's respecting the theme color